### PR TITLE
Update to 0.7.4

### PR DESCRIPTION
**Pull request by Luthaf**
And activate windows build on the side! 🎉

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,35 +11,27 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win]
+  skip: true  # [win and py<3.5]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Not that it is needed, but could do `py2k` instead of `py<3.5`.

Comment (jakirkham) [reply recipe/meta.yaml]: Though it does need features.

```yaml
    features:
        - vc9   # [win and py27]
        - vc14  # [win and py35]
        - vc14  # [win and py36]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,35 +11,27 @@ source:
...
 requirements:
   build:
     - cmake
-    - libnetcdf 4.4.*
     - toolchain
-  run:
-    - libnetcdf 4.4.*
+    # We need to depend on Python to be sure to have MSVC 14 available for the
+    # build: https://github.com/conda/conda/wiki/VC-features
+    - python  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: This can be solved more cleanly with the `vc` features. Also can use the `vc` package if you like.

Comment (Luthaf) [reply recipe/meta.yaml]: I'll try again, but the last time I gave it a look it did not worked with only the `features` entry.

Comment (jakirkham) [reply recipe/meta.yaml]: What part wasn't working?

Comment (Luthaf) [reply recipe/meta.yaml]: I don't remember, it was long ago. It seems to work in a VM, so I made the changes. Thanks for the suggestion.

Comment (Luthaf) [reply recipe/meta.yaml]: OK, I should have let the build finish ... I get this error locally:

```
Can't build ...\chemfiles-lib-feedstock\recipe due to unsatisfiable dependencies:
The following specifications were found to be in conflict:
    - chemfiles-lib 0.7.4 vc14_0 -> feature:vc14 
Use "conda info <package>" to see the dependencies for each package.
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,35 +11,26 @@ source:
...
-    # run the tests
-    - cp $RECIPE_DIR/27d89caa55e1f42cdbbed5a322b51c702066d871.tar.gz tests/
-    - cmake -DCHFL_BUILD_TESTS=ON .
-    - cmake --build . --config release
-    - ctest --output-on-failure
+  features:
+    - vc14  # [win]
 
 requirements:
   build:
```
Comment (jakirkham) [inline recipe/meta.yaml:20]: Still need `python` in `build`. Just not `run`.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -11,35 +11,26 @@ source:
...
     - cmake
-    - libnetcdf 4.4.*
     - toolchain
-  run:
-    - libnetcdf 4.4.*
```
Comment (jakirkham) [inline recipe/meta.yaml]: Either need to add the `vc` package to `run` (and optionally `build`).

File: recipe/meta.yaml [hunk 5]
```diff
@@ -11,35 +11,26 @@ source:
...
 
 test:
```
Comment (jakirkham) [inline recipe/meta.yaml]: ...or need to add...

```yaml
  requires:
    - python 2.7.*  # [win and py27]
    - python 3.4.*  # [win and py34]
    - python 3.5.*  # [win and py35]
```

**Review by jakirkham — COMMENTED** [review]
Added notes about configuring the Windows build below.

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Right so added some notes below as follow-up.

Comment (Luthaf) [reply recipe/meta.yaml]: OK, thanks! I'll try this tomorrow. The doc could use an example for this =)

**Comment by Luthaf** [comment]
This seems to work. If you don't have any more remarks, I'll merge this!

And thank you again for your help, it is very appreciated =)

File: recipe/meta.yaml [hunk 6]
```diff
@@ -11,35 +11,28 @@ source:
...
+    - vc14  # [win]
 
 requirements:
   build:
     - cmake
-    - libnetcdf 4.4.*
     - toolchain
   run:
-    - libnetcdf 4.4.*
+    - vc  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Would pin this to `14` exactly.

**Comment by jakirkham** [comment]
Nice work @Luthaf! One minor adjustment above. Otherwise LGTM.

**Pull request merged**